### DISCUSSION TITLE
Does yolo have a useful annotation GUI?

### DISCUSSION THREAD

**Discussion by doubletaro** [ultralytics/yolov5#7944] [Q&A]
Hello everyone.
I'm using yolov5.
However, since it is awkward to annotate target images, I wondered if I could make a learning data just by fine-tuning the result output by the prototype AI model.
Is there an annotation tool for yolov5 like the one above?

**Comment by glenn-jocher** [comment]
@doubletaro you can generate txt labels for images with detect.py and val.py using --save-txt, and --save-hybrid with val.py

**Comment by doubletaro** [comment]
@glenn-jocher 
I tried it!
Thanks to your advice, I feel like I can do what I want to do.
However, the "LabelImg" I'm using couldn't read the output in this txt format.
Is there a GUI tool that can read this txt format output?
![0001](https://user-images.githubusercontent.com/45992024/170403685-e5cc3598-22ba-4822-9c19-a43200375485.png)

**Reply by glenn-jocher** [reply]
@doubletaro Ultralytics HUB (https://hub.ultralytics.com/) can visualize your YOLOv5 datasets, but does not have annotation tools yet. For annotation you might want to try [Roboflow](https://roboflow.com/?ref=ultralytics)

**Reply by doubletaro** [reply]
@glenn-jocher 
Thank you for the Information!
I will try using Roboflow.
If not ideal, I'll  consider designing GUI.
